study.md

```diff
@@ -34,7 +34,7 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+Map and Select. An array is one method, while an enumerable is a bunch of methods packaged together. This would explain the definition difference.
```

Comment:
This does not answer the question.

`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.